server/tserver/src/main/java/org/apache/accumulo/tserver/NativeMap.java

```diff
@@ -70,11 +70,10 @@
   static {
     // Check standard directories
     List<File> directories = new ArrayList<>(Arrays.asList(new File[] {new File("/usr/lib64"), new File("/usr/lib")}));
-    // Check in ACCUMULO_HOME location, too
-    String accumuloHome = System.getenv("ACCUMULO_HOME");
-    if (accumuloHome != null) {
-      directories.add(new File(accumuloHome + "/lib/native"));
-      directories.add(new File(accumuloHome + "/lib/native/map")); // old location, just in case somebody puts it here
```

Comment:
+1, but noting that "WARN + EXIT" behavior is essentially the definition of a "FATAL" error.